eng/Analyzers_ShippingRules.ruleset

```diff
@@ -1,41 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Common diagnostic rules for all Roslyn Analyzer projects"
-         Description="This file contains diagnostic settings used by all Roslyn Analyzer projects. Projects that need specific settings should have their own rule set files that Include this one, and then make the necessary adjustments." 
-         ToolsVersion="14.0">
-
-  <Rules AnalyzerId="Microsoft.ApiDesignGuidelines.Analyzers" RuleNamespace="Microsoft.ApiDesignGuidelines.Analyzers">
-    <!-- ValidateArgumentsOfPublicMethods - only useful for libraries with supported public API surface, we don't have any -->
-    <Rule Id="CA1062" Action="None" />
```

Comment:
I see that you moved few entries from here to the common globalconfig but lot of the entries were not moved and just deleted. I wanted to make sure that the globalconfigs you added are indeed being respected by the build, because I would have expected dropping some of these entries completely to lead to build errors.